### ISSUE TITLE
LICENSE incompatibility 

### ISSUE THREAD

**Issue by ludwiglierhammer** [glamod/glamod-marine-processing#10]
[FOSSA](https://app.fossa.com/projects/git%2Bgithub.com%2Fglamod%2Fglamod-marine-processing/refs/branch/master/9b9a74bc0a0f2daceb6338729c804c28fedeaeee/issues/licensing?page=1&count=20&sort=issue_count_desc&grouping=revision&status=active&filter%5Btype%5D%5B0%5D=policy_flag&revisionScanId=58920282) detected a LICENSE issue with [Levenshtein](https://pypi.org/project/Levenshtein/):

These packages contain code files that may require you to disclose your source code under a compatible license, unless they’re distributed and run as completely separate processes & packages.

```
Levenshtein is free software; you can redistribute it and/or modify it
under the terms of the GNU General Public License as published by the Free
Software Foundation; either version 2 of the License, or (at your option)
any later version.
```

@aanderss: Any idea how to solve this issue? I am not yet a licensing expert ;)

**Comment by ludwiglierhammer** [comment]
[Here](https://github.com/rapidfuzz/Levenshtein/blob/main/COPYING) you can find the **GNU_General_Public_License** license used in Levenshtein.

**Comment by ludwiglierhammer** [comment]
The Levenshtein package for calculating the [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance) is uised in the metadata_suite script [merge_countries](https://github.com/glamod/glamod-marine-processing/blob/master/glamod_marine_processing/metadata_suite/scripts/merge_countries.py).

**Comment by ludwiglierhammer** [comment]
This package is used in metadata_suite.
In PR #16 we deleted metadata_suite because it is not used and/or fixed for release 7.0.
@aanderss: Problem solved ?

**Comment by ludwiglierhammer** [comment]
Problem solved: see above comment
